CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## Unreleased
 
+## 1.173.1 (May 21, 2024)
+ENHANCEMENTS:
+* resource/spotinst_ocean_aks_np: Added support for `tasks` object under `scheduling` block.
```

Comment:
modify the sentence to:  Added `tasks` object under `scheduling` block supporting schedule roll.

Reply:
Modified